.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@
 
 <!-- Add additional detailed description of need for change if no related issue -->
 
-<!-- If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/.github/CONTRIBUTING.md -->
+**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/.github/CONTRIBUTING.md
```

Comment:
just notice that the branch is `main` instead of `master` so that the url path should be `/main/`

Reply:
good catch! thought I had fixed that previously.

Reply:
fixed